src/services/supabase.ts

```diff
@@ -21,6 +21,10 @@ const supabaseSettings = {
     anonKey: process.env.REACT_APP_SUPABASE_ANON_KEY,
 };
 
+// Related to the AUth stuff
+export const supabaseAuthError = 'error_description';
+export const userNotFound = 'User not found';
```

Comment:
I want to find a better way to know these... might look into using the library from Supabase directly? Not sure yet.